aviary/subsystems/propulsion/test/test_custom_engine_model.py

```diff
@@ -225,34 +204,6 @@ def test_custom_engine(self):
 
         assert_near_equal(float(prob.get_val('traj.cruise.rhs_all.y')), 4., tol)
 
-        prob_vars = prob.list_problem_vars(
-            print_arrays=True, driver_scaling=False, out_stream=None)
-
-        design_vars_dict = dict(prob_vars['design_vars'])
-
-        # List of all expected variable names in design_vars
-        expected_var_names = [
-            'traj.phases.cruise.indep_states.states:altitude',
-            'traj.phases.cruise.indep_states.states:velocity',
-            'traj.phases.cruise.indep_states.states:mass',
-            'traj.phases.cruise.indep_states.states:distance',
-        ]
-
-        # Check that all expected variable names are present in design_vars
-        for var_name in expected_var_names:
-            self.assertIn(var_name, design_vars_dict)
-
-        # Check values
```

Comment:
Why were these asserts all removed?

Reply:
Not important for testing if the custom engine model was used correctly; I had to update their names and realized there was no reason for them to be there